pkgs/development/python-modules/litemapy/default.nix

```diff
@@ -3,21 +3,25 @@
   buildPythonPackage,
   fetchFromGitHub,
   nbtlib,
+  typing-extensions,
 }:
 
 buildPythonPackage rec {
   pname = "litemapy";
-  version = "0.7.2b0";
+  version = "0.10.0b0";
   format = "setuptools";
```

Comment:
Can you migrate to `format = "pyproject"` with `build-system = [ setuptools ]`

Reply:
I am not so sure about that... 
Correct me if i am wrong but ist't `format ? "pyproject"` for projects with a `pyproject.toml`?
As far as i can see the project is still using the old structure.

Reply:
alright, i stand corrected according to [buildPythonPackage](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-function-buildpythonpackage-function).
I will use
```nix
pyproject = true;
build-system = [ setuptools ];
```

Reply:
@Sigmanificient this should be resolved now